src/libsyntax_expand/lib.rs

```diff
@@ -33,6 +33,25 @@ pub use mbe::macro_rules::compile_declarative_macro;
 pub mod base;
 pub mod build;
 pub mod expand;
+#[macro_use] pub mod config;
 pub mod proc_macro;
 
 crate mod mbe;
+
+#[cfg(test)]
```

Comment:
Could you add a comment telling why all these tests are here.

Reply:
Great idea! and done :)